### PR TITLE
[chrome] Add definition for chrome.runtime.OnInstalledReason

### PR DESCRIPTION
**Pull request by isnullxbh**
Change the type of the 'reason' (InstalledDetails) to OnInstalledReason according
to the https://developer.chrome.com/docs/extensions/reference/runtime/#event-onInstalled.

Please fill in this template.

- [x] Use a meaningful title for the pull request. Include the name of the package modified.
- [ ] Test the change in your own code. (Compile and run.)
- [ ] [Add or edit tests](https://github.com/DefinitelyTyped/DefinitelyTyped/blob/master/README.md#my-package-teststs) to reflect the change.
- [x] Follow the advice from the [readme](https://github.com/DefinitelyTyped/DefinitelyTyped/blob/master/README.md#make-a-pull-request).
- [x] Avoid [common mistakes](https://github.com/DefinitelyTyped/DefinitelyTyped/blob/master/README.md#common-mistakes).
- [x] [Run `npm test <package to test>`](https://github.com/DefinitelyTyped/DefinitelyTyped/blob/master/README.md#running-tests).

Select one of these and delete the others:

If adding a new definition:
- [ ] The package does not already provide its own types, or cannot have its `.d.ts` files generated via `--declaration`
- [ ] If this is for an npm package, match the name. If not, do not conflict with the name of an npm package.
- [ ] Create it with `dts-gen --dt`, not by basing it on an existing project.
- [ ] Represents shape of module/library [correctly](https://www.typescriptlang.org/docs/handbook/declaration-files/library-structures.html)
- [ ] `tslint.json` [should contain](https://github.com/DefinitelyTyped/DefinitelyTyped/blob/master/README.md#linter-tslintjson) `{ "extends": "dtslint/dt.json" }`, and no additional rules.
- [ ] `tsconfig.json` [should have](https://github.com/DefinitelyTyped/DefinitelyTyped/blob/master/README.md#tsconfigjson) `noImplicitAny`, `noImplicitThis`, `strictNullChecks`, and `strictFunctionTypes` set to `true`.

If changing an existing definition:
- [x] Provide a URL to documentation or source code which provides context for the suggested changes: <<url here>>
- [ ] If this PR brings the type definitions up to date with a new version of the JS library, update the version number in the header.

If removing a declaration:
- [ ] If a package was never on Definitely Typed, you don't need to do anything. (If you wrote a package and provided types, you don't need to register it with us.)
- [ ] Delete the package's directory.
- [ ] Add it to `notNeededPackages.json`.

### CODE REVIEW

**Review by nicolas377 — APPROVED** [review]
looks good to me! modifying tests here won't help, so we'll have to wait on a maintainer to approve this.

**Comment by nicolas377** [comment]
If we could get some more chrome owners to check this out and approve this, it should be enough to get a DT maintainer to see this PR.

**Review by nicolas377 — CHANGES_REQUESTED** [review]
Now that I'm looking at this a second time, I see there aren't any tests to cover onInstalled events in the first place. @isnullxbh could you add some tests for that? If you want, I can PR some tests to your fork.

**Comment by isnullxbh** [comment]
@nicolas377, yes, sure. I'll do it a bit later.

>  If you want, I can PR some tests to your fork
I think I need help with test cases. Can you suggest me few cases for unit tests?

**Comment by nicolas377** [comment]
@isnullxbh do you still want to be an owner? i'm making a pr with tests for this, and I can add you as an owner if you want.

**Comment by isnullxbh** [comment]
Hello, @nicolas377! No, but thanks for the suggestion.

**Pull request closed without merging**
